uber/model_checks.py

```diff
@@ -74,7 +74,7 @@ def group_money(group):
         amount_refunded = int(float(group.amount_refunded))
```

Comment:
Just out of curiosity, why do we do the `int(float(group.amount_refunded))` cast?